packages/flutter/lib/src/cupertino/app.dart

```diff
@@ -651,6 +652,12 @@ class _CupertinoAppState extends State<CupertinoApp> {
   Widget build(BuildContext context) {
     final CupertinoThemeData effectiveThemeData = (widget.theme ?? const CupertinoThemeData())
         .resolveFrom(context);
+    final Brightness brightness =
```

Comment:
A comment here similar to what Mitchell said in https://github.com/flutter/flutter/pull/162297/files#r1938130383 could help understandability. 

```suggestion
    // Prefer theme brightness if set, otherwise check system brightness. 
    final Brightness brightness =
```

Reply:
done